versioned_docs/version-8.1/reference/regions.md

```diff
@@ -20,5 +20,6 @@ Running on a Trial and want to try a different region, or interested in other re
 - US East: us-east1
 - US Central: us-central1
 - Australia Southeast: australia-southeast1
+- North America Northeast: northamerica-northeast2
```

Comment:
probably this has to be on 8.2. But I don't know the release process tbh

Reply:
from my point of view we could release new regions immediately, let's not wait for minor release